### PR TITLE
Grade Team Open Response Assessments (ORAs)

### PR DESCRIPTION
**Pull request by nsprenkle**
## [EDUCATOR-5115](https://openedx.atlassian.net/browse/EDUCATOR-5115)

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

To be released with the Team Assignments feature

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 1]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+=======================================
+For Team Open Assessments
+=======================================
+
+Since team-based open response assessments can only be assessed by staff,
+course team members see a **Grade Available Responses** option at the end of
+the assignment in the course, and learners see a **Staff Assessment** step in
+their assignment.
+
+Submitting a staff assessment for a team ORA has the following results.
```
Comment (muselesscreator) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: This line should probably end in a **:** ("the following results:")
Alternate Proposal: 
**[Team member grades]** Scores submitted in `Staff Assessment` for Team ORAs are assigned to all members of the given team.

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 2]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+
+* The score that you give a team in a staff assessment assigns the stame score
+  to all members of the given team.
+
+To perform a staff assessment in a team assignment, follow these steps.
```
Comment (muselesscreator) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: "follow these steps:"

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 3]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+
+#. View the live version of your course in the LMS, and then go to the ORA
```
Comment (muselesscreator) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: "and then go to" => "and select"
Implies less looking and better indicates that you should be able to access it directly from the top-level course-view.

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 4]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+   assignment.
+
+#. Scroll to the end of the problem, and then select **Grade Available
+   Responses**.
+
+   In the dialog that opens, the number of available and checked out responses
+   is shown. Checked out responses are responses that are currently being
```
Comment (muselesscreator) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: "Checked out responses" should be formatted.

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 5]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+   graded by you or another course team member.
+
+#. Select the **Staff Assessment** heading to open a response that is
+   available for grading.
+
+#. Perform an evaluation of the response using the problem's rubric.
+
+#. Select **Submit assessment** to submit the assessment. The rubic will prompt
```
Comment (muselesscreator) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: "rubic" => "rubric"

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 6]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+   you that the grade will apply to all members of the team and ask if you want
+   to continue. Confirm to apply the grade and and close the grading dialog.
```
Comment (muselesscreator) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: "The rubric will prompt you to confirm that the grade selected will apply to all members of the before continuing."?

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 7]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+=======================================
+
+Since team-based open response assessments can only be assessed by staff,
+course team members see a **Grade Available Responses** option at the end of
+the assignment in the course, and learners see a **Staff Assessment** step in
+their assignment.
+
+Submitting a staff assessment for a team ORA has the following results.
+
+* The score that you give a team in a staff assessment assigns the stame score
```
Comment (jansenk) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: s/stame/same

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst (hunk 3 above)
Comment (jansenk) [reply en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: I would suggest "and navigate to"

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 8]
```diff
@@ -133,6 +133,45 @@ To perform a staff assessment in an assignment, follow these steps.
...
+  to all members of the given team.
+
+To perform a staff assessment in a team assignment, follow these steps.
+
+#. View the live version of your course in the LMS, and then go to the ORA
+   assignment.
+
+#. Scroll to the end of the problem, and then select **Grade Available
```
Comment (jansenk) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: perhaps s/the end of the problem/the bottom of the component? I think bottom is better than "end" but i'm not sure what word we should use to describe the rendered xblock

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst (hunk 4 above)
Comment (jansenk) [reply en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: why?

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst (hunk 6 above)
Comment (jansenk) [reply en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: "You will receive an alert to notify you that the grade selected will apply to all members of the team"?

Comment (jansenk) [reply en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: I think the word "prompt" implies some sort of input beyond just clicking "okay"

**Comment by jansenk** [comment]
Why do my replies to other comments show up both as replies to comments but then also as contextless comments within my review block? Why couldn't github have fixed that instead of making everything round

**Comment by nsprenkle** [comment]
RE: @jansenk and @muselesscreator, some of the wording critiques are carry-overs from the non-team versions of the documentation.

I'm okay to update those with the understanding that it will require propagating those changes to other areas of the documentation.

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst (hunk 4 above)
Comment (nsprenkle) [reply en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: I disagree since it is not, verbatim, an element on the page

**Comment by nsprenkle** [comment]
@jansenk and @muselesscreator, I modified the docs to just note the deviations in the team process from the individual process. This, hopefully, also crossed off some of the other comments and could use re-reveiew.

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 9]
```diff
@@ -133,6 +133,23 @@ To perform a staff assessment in an assignment, follow these steps.
...
+For Team Open Assessments
+=======================================
+
+Team-based open response assessments can only be assessed by staff. Learners
+will see this as a **Staff Assessment** step in their assignment. Course team
+members will see a **Grade Available Responses** option at the end of the
+assignment.
+
+.. note:: Staff assessments for team ORAs follow the same process as for individual ORAs
+   except that scores are applied to all members of the submitting team.
```
Comment (Andytr1) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: Does it make sense to be explicit here that the `same score` will be applied to all team members?

**Review by Andytr1 — APPROVED** [review]

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst (hunk 9 above)
Comment (nsprenkle) [reply en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: Good point. Updated wording for better clarity

**Comment by nsprenkle** [comment]
Closing to do review Google Drive ([link](https://docs.google.com/document/d/1_5Nz2YzZVSzm2V7PwWWmyT4qDlFUAioxvGSbKBQP0UM/edit)). Will move changes back here once ready for release

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 10]
```diff
@@ -133,6 +133,24 @@ To perform a staff assessment in an assignment, follow these steps.
    dialog. Alternatively, select **Submit assessment and continue grading** to
    submit the assessment and immediately grade another submission.
 
+=======================================
+For Team Open Assessments
```
Comment (nsprenkle) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst:137]: Should be For Team Open Response Assessments

**Pull request closed without merging**